README.md

```diff
@@ -45,6 +45,11 @@ Reference: [Maven Settings > Plugin Repositories](http://maven.apache.org/settin
 
 Reference: [Maven Settings > Repositories](http://maven.apache.org/settings.html#Plugin_Repositories)
 
+### `plugin_groups`
+**Optional** json array of plugin groups to add to settings.xml
+
+Reference: [Maven Settings > Profiles](http://maven.apache.org/settings.html#Plugin_Groups)
```

Comment:
Change the link text to `Maven Settings > Plugin Groups`

Reply:
Fixed. Sorry, bad copy-paste :-/